### DISCUSSION TITLE
alit开了又拍云的cdn，不能连接dav了

### DISCUSSION THREAD

**Discussion by starssr** [AlistGo/alist#7165] [Q&A] [answered]
又拍云设置如下，后台和其他都没有问题，就是dav无法连接
![1726030023590](https://github.com/user-attachments/assets/2b861baf-09fd-42a3-b8f8-5cca497a3ae5)
![1726030057057](https://github.com/user-attachments/assets/c081621b-ad2d-445e-af3a-bdcb6018fa5a)
![1726030072471](https://github.com/user-attachments/assets/cee87b9d-b62f-4758-9bc9-5d23cfa41c7b)
![1726030098130](https://github.com/user-attachments/assets/e2954653-a992-4e44-b9d4-2b58f8bae5e8)

**Comment by starssr** [comment]
没有遇到同样问题的兄弟吗

**Comment by xhofe** [comment, marked as answer]
又拍云不支持 webdav 扩展方法。

**Reply by starssr** [reply]
我把dav目录选择不缓存，回源跟随也不行吗，绕不过cdn哈？大佬知道哪个cdn可以呢，目前用的是又拍云，多吉云，还有缤纷云
